src/components/list/list.ios.scss

```diff
@@ -191,4 +191,11 @@ $list-ios-header-background-color:    transparent !default;
     color: $color-contrast;
     background-color: $color-base;
   }
+}
+
+// iOS List inputs
+// --------------------------------------------------
+
+.list-ios .item-input:last-child {
+  border-bottom: 1px solid $list-ios-border-color;
```

Comment:
iOS should use `$hairlines-width` not `1px`, see: https://github.com/driftyco/ionic/blob/master/src/components/list/list.ios.scss#L66